osu.Game/Overlays/Profile/Header/PreviousUsernamesContainer.cs

```diff
@@ -0,0 +1,181 @@
+                RelativeSizeAxes = Axes.X;
+                Masking = true;
+                AlwaysPresent = true;
+                CornerRadius = 5;
+                Children = new Drawable[]
+                {
+                    background = new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                    },
+                    new FillFlowContainer
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Direction = FillDirection.Vertical,
+                        Spacing = new Vector2(0, 5),
+                        Children = new Drawable[]
+                        {
+                            new Container
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                Height = header_height,
+                                Children = new Drawable[]
+                                {
+                                    new SpriteText
+                                    {
+                                        Anchor = Anchor.BottomLeft,
+                                        Origin = Anchor.BottomLeft,
+                                        Padding = new MarginPadding { Left = content_padding },
+                                        Text = @"formerly known as",
+                                        Font = OsuFont.GetFont(size: 14, italics: true)
+                                    }
+                                }
+                            },
+                            Usernames = new TextFlowContainer
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Full,
+                                Spacing = new Vector2(5, 5),
+                                Padding = new MarginPadding { Left = content_padding, Bottom = margin },
+                            },
+                        }
+                    }
+                };
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours)
+            {
+                background.Colour = colours.GreySeafoamDarker;
+            }
+
+            public override void Clear(bool disposeChildren) => Usernames.Clear(disposeChildren);
+
+            protected override void PopIn() => this.FadeIn(duration, Easing.OutQuint);
+
+            protected override void PopOut() => this.FadeOut(duration, Easing.OutQuint);
+        }
+
+        private class HoverIconContainer : Container
+        {
+            public Action ActivateHover;
+
+            public HoverIconContainer()
+            {
+                AutoSizeAxes = Axes.Both;
+                Child = new SpriteIcon
+                {
+                    Margin = new MarginPadding(margin) { Top = 6 },
```

Comment:
Why the `Top = 6`?

Reply:
For visual. That icon is not a perfect square and the size of the SpriteIcon is not equal the size of the icon itself